float32 leftMotorAmps
float32 rightMotorAmps